public/js/addLocation.js

```diff
@@ -0,0 +1,72 @@
+$(document).ready(function() {
+    $('.sidenav').sidenav();
+    $(".dropdown-trigger").dropdown( {
+        hover: true
+    });
+  });
+
+
+
+const labels = '123456789';
+let labelIndex = 0;
+let map, infoWindow;
+function initMap() {
+    map = new google.maps.Map(document.getElementById('map'), {
+        center: { lat: 50, lng: -100 },
+        zoom: 6
+    });
+    infoWindow = new google.maps.InfoWindow;
+
+    map.addListener('click', function (e) {
+        placeMarker(e.latLng, map);
+        let newLocation = {
+            locationName: "test",
+            latitude: e.latLng.lat,
+            longitude: e.latLng.lng,
+            radius: 100
+        }
+
+        $.post("/api/locations", newLocation).then(function (data) {
+            console.log(data);
+        });
+
+    });
+
+    $("#findMe").on("click", function () {
+        // Try HTML5 geolocation.
+        if (navigator.geolocation) {
+            navigator.geolocation.getCurrentPosition(function (position) {
```

Comment:
I recommend if you are not going to make this an arrow function to name any anonymous function, it isn't necessary but it helps stack traces for errors be a little more readable. e.g.
navigator.geolocation.getCurrentPosition(function positionSuccess(position) {
if you make these unique names it is also easy to find if you see one in a stack trace (that being said there's always line numbers in stack traces too)